package.json

```diff
@@ -129,6 +129,7 @@
     "react-dom": "^16.3.0-alpha.1",
     "react-native-storybook-loader": "^1.6.0",
     "react-native-typescript-transformer": "^1.2.3",
+    "react-relay-network-modern": "^2.5.1",
```

Comment:
Rad, so this was pretty straightforward to drop in

Reply:
Yup it was pretty straightforward aside from the cache middleware